scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -152,6 +153,23 @@
                                      @(:pool-name->fenzo-state-atom compute-cluster)
                                      mesos-status))
 
+(defn write-status-to-passport
+  "Helper function for logging completed job status to cook passport"
+  [{:keys [name]}
+   {:keys [task-id state reason]}]
+  (let [pod-name (task-id :value)
+        event-map {:pod-name pod-name
+                   :cluster-name name
+                   :state state
+                   :reason reason
+                   :event-type passport/pod-completed}
+        instance-uuid (api/pod-name->instance-uuid pod-name)
+        job-uuid (api/pod-name->job-uuid pod-name)
+        event-map (cond
+                    instance-uuid (assoc event-map :instance-uuid instance-uuid)
+                    job-uuid (assoc event-map :job-uuid job-uuid))]
```

Comment:
Should we make this a helper function since it's being done in a couple places already?

Reply:
Yeah good call, it's the exact same in both spots. I might add it as a helper function in the `passport` namespace.

Reply:
actually going to keep it in the `kubernetes.api` namespace for consistency.